app/js/daemonManager.js

```diff
@@ -61,6 +61,9 @@ function DaemonManager() {
 		});
 	}
 
+	/**
+	 * Checks if there is an update available
```

Comment:
I thought the same thing when I heard about the go standard. But you usually also get a bit of helpful text explaining the function, which is nice.